invisible_cities/cities/irene_test.py

```diff
@@ -70,6 +70,12 @@ def test_command_line_irene_electrons_40keV(conf_file_name_mc, config_tmpdir, IC
             mctracks_out = h5out.root.MC.MCTracks[0]
             np.testing.assert_array_equal(mctracks_in, mctracks_out)
 
+            # chech events numbers & timestamps
+            evts_in = h5in.root.Run.events[:3]
```

Comment:
Could we get rid of these magic numbers? There are a couple of 3s here, and a couple of 0s a bit higher up.